specs/subresourceintegrity/index.html

```diff
@@ -146,7 +146,7 @@ <h2 id="introduction">Introduction</h2>
         integrity="sha256-C6CB9UYIS9UJeqinPHWTHVqh/E1uhG5Twh+Y5qFQmYg="&gt;
 </code></pre>
 
-  <p class="example highlight">Scripts, of course, are not the only resource type which would benefit
```

Comment:
Are you not using the Makefile or do you have a different version of Kramdown? I'm on version 1.6.0, which keeps this attribute in place.